### DISCUSSION TITLE
Validator constraint similar to array_is_list

### DISCUSSION THREAD

**Discussion by alcohol** [symfony/symfony#54738] [Q&A] [answered]
Just curious, is there a validator constraint that is basically the equivalent of `array_is_list`? I noticed `Type` does not support it (yet?). And I could not quite figure out how to get `Collection` to assert the same constraints as `array_is_list` does, even when combined with `All`.

Edit: my current solution is to use a `Callback` which simply tests if `array_is_list($value)` :sweat_smile:

**Comment by mbabker** [comment, marked as answer]
This will be possible with `#[Assert\Type('list')]` in Symfony 7.1 thanks to https://github.com/symfony/symfony/pull/52954
